lithops/standalone/standalone.py

```diff
@@ -264,6 +325,11 @@ def create_workers(workers_to_create):
 
         self.jobs.append(job_payload['job_key'])
 
+        # in case workers policy is strict, track all required workers create
+        if self.workers_policy == 'strict':
+            threading.Thread(target=wait_workers_ready, args=(new_workers,), daemon=True).start()
```

Comment:
Maybe we should also check here the execution_mode is not `consume` mode? consume_mode does not have new_workers so if worker_policy is strict it will fail

Reply:
You are right. new_workers wouldn't be defined at that point in case of consume. Fixed.